### PR TITLE
Overriding repository for analytic specific data

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php [hunk 1]
```diff
@@ -2,13 +2,96 @@
...
+    public function __construct(EntityManager $em, ClassMetadata $class)
+    {
+        $class->setCustomRepositoryClass(__CLASS__);
+        parent::__construct($em, $class);
+    }
+
+    /**
+     * @param \DateTime $startDate
+     * @param \DateTime $endDate
+     * @return mixed
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php]: Mixed? Would it not be a `Questionnaire` Entity?

File: src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php [hunk 2]
```diff
@@ -2,13 +2,96 @@
...
+        return $query
+            ->setParameter(':start_date', $startDate->format(\DateTime::ISO8601))
+            ->setParameter(':end_date', $endDate->format(\DateTime::ISO8601))
+            ->getResult();
+    }
+
+    /**
+     * @param \DateTime $startDate
+     * @param \DateTime $endDate
+     * @return mixed
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Repository/QuestionnaireRepository.php]: Same here, mixed?

File: src/TransformCore/PHE/AnalyticsBundle/Service/QuestionnaireService.php [hunk 3]
```diff
@@ -10,5 +15,60 @@
...
+     */
+    public function getRepository()
+    {
+        return $this->repository;
+    }
+
+    /**
+     * @param \DateTime $startDate
+     * @param \DateTime $endDate
+     * @return mixed
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Service/QuestionnaireService.php]: Mixed?

Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Service/QuestionnaireService.php]: Noted, let me address in next PR, came from the doctrine documentation

**Pull request merged**